docs/how-to-install.md

```diff
@@ -125,6 +135,17 @@ variable in the `conf/gravitino-env.sh` file. Then create a `Remote JVM Debug`
 configuration in `IntelliJ IDEA` and debug `gravitino.server.main`.
 :::
 
+#### Manage Gravitino Iceberg REST server in Gravitino package
+
+You can operate the Iceberg REST server either as a standalone server or as an auxiliary service within the Gravitino server. There are some notable differences between these two modes:
```

Comment:
"You can run Iceberg REST server...".

Reply:
Also you mentioned that "There are some notable differences", so what is the difference here? what you mentioned here is not the difference, is the step to configure and run.

So I suggest you to not use table, just list them step by step to tell user how to run in two modes.

Reply:
done